packages/recomponents/src/components/r-popper/r-popper.scss

```diff
@@ -15,14 +15,14 @@
     box-sizing: border-box;
     display: inline-block;
     position: absolute;
-    border-radius: var(--popper-border-radius);
-    z-index: var(--popper-z-index);
-    box-shadow: var(--popper-box-shadow);
-    background: var(--popper-background);
-    min-width: var(--popper-min-width);
+    border-radius: var(--popover-border-radius);
```

Comment:
Why is it called `--popover`? The component is called `popper`, class - `popper` too.

Reply:
`popper` and `popover`, I think the `popover` is the correct word? Anyway, I agree we always should be consistent.

BTW, I fixed the bug: `some of CSS vars are not defined but are used`, not the naming issue.